lib/app/components/pages/talk_room/views/talk_room_view.dart

```diff
@@ -17,11 +17,18 @@ class TalkRoomView extends GetView<TalkRoomController> {
         leading: const atoms.BackButton(),
         backgroundColor: Colors.pink,
       ),
-      body: const Center(
-        child: Text(
-          'TalkRoomView is working',
-          style: TextStyle(fontSize: 20),
-        ),
+      body: Column(
+        children: [
+          Expanded(
+            child: Column(
+              children: const [
+                //FIXME: ここはメッセージバルーンのListView.builderに置き換わる
+                Text('ここはメッセージバルーンのListView.builderに置き換わる'),
+              ],
+            ),
+          ),
+          const MessageBar(),
+        ],
       ),
```

Comment:
```suggestion
      body:Stack(
        children: const [
          SingleChildScrollView(
            child: SafeArea(
              child: Text('ここはメッセージバルーンのListView.builderに置き換わる'),
            ),
          ),
          MessageBar(),
        ],
      ),
```
`Column`で上から配置していくよりも`Stack`で重ねる方がいい気がするので、
`Stack`にしましょう

Reply:
ご指摘の通り修正しました。